packages/owl-plplot/owl-plplot.1.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "Liang Wang <ryanrhymes@gmail.com>"
+authors: [ "Liang Wang" ]
+license: "MIT"
+homepage: "https://github.com/owlbarn/owl-plplot"
+dev-repo: "git+https://github.com/owlbarn/owl-plplot.git"
+bug-reports: "https://github.com/owlbarn/owl-plplot/issues"
+doc: "https://owlbarn.github.io/owl-plplot/"
+synopsis: "Owl-Plplot: thin plotting layer for Owl based on plplot"
+
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" {>= "4.10.0"}
+  "dune" {>= "2.0.0"}
+  "owl" {>= "1.0"}
+  "plplot"
```

Comment:
Note that the ocaml plplot library is no longer maintained, and this will not compile in recent distributions. It will restart compiling if plplot gets updated though, so that is surely not a blocker